main/src/ca/uwaterloo/flix/language/phase/Resolver.scala

```diff
@@ -1721,6 +1721,19 @@ object Resolver {
           mapN(esVal) {
             es => ResolvedAst.Pattern.Tuple(es, loc)
           }
+
+        case NamedAst.Pattern.Record(pats, pat, loc) =>
+          val psVal = traverse(pats) {
+            case NamedAst.Pattern.Record.RecordFieldPattern(field, tpe, pat1, loc1) =>
+              // Unsure about Wildness
+              mapN(traverseOpt(tpe)(resolveType(_, Wildness.ForbidWild, env, taenv, ns0, root)), traverseOpt(pat1)(visit)) {
```

Comment:
What should the value of `Wildness` be in `resolveType` when resolving a type annotation in a pattern match on a field? e.g.
```scala
match { x = 1 } {
    case { x : Int32 = 1 } => ??? // Consider `Int32`
}
```

Reply:
It seems like wildness should be allowed, IMO.

Reply:
What does Wildness refer to? Kinds are in the next phase, correct? So what is Wildness here?